airbyte-integrations/connectors/source-google-ads/source_google_ads/manifest.yaml

```diff
@@ -0,0 +1,808 @@
+      paginator:
+        type: NoPagination
+      record_selector:
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - results
+        schema_normalization: Default
+      partition_router:
+        type: GroupingPartitionRouter
+        group_size: 1
+        deduplicate: true
+        underlying_partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              stream: "#/definitions/customer_client_non_manager"
+              parent_key: "clientCustomer"
+              partition_field: "customer_id"
+    transformations:
+      - type: KeysToSnakeCase
+      - type: DpathFlattenFields
+        field_path: ["campaign"]
+        delete_origin_value: True
+        key_transformation:
+          type: KeyTransformation
+          prefix: "campaign."
+      - type: AddFields
+        fields:
+          - path: ["campaign.url_custom_parameters"]
+            value: "{{ record['campaign.url_custom_parameters'] | map('tojson') | list | tojson | safe }}"
+      - type: DpathFlattenFields
+        field_path: ["campaign.network_settings"]
+        delete_origin_value: True
+        key_transformation:
+          type: KeyTransformation
+          prefix: "campaign.network_settings."
+      - type: DpathFlattenFields
+        field_path: ["campaign.geo_target_type_setting"]
+        delete_origin_value: True
+        key_transformation:
+          type: KeyTransformation
+          prefix: "campaign.geo_target_type_setting."
+      - type: DpathFlattenFields
+        field_path: ["metrics"]
+        delete_origin_value: True
+        key_transformation:
+          type: KeyTransformation
+          prefix: "metrics."
+      - type: DpathFlattenFields
+        field_path: ["segments"]
+        delete_origin_value: True
+        key_transformation:
+          type: KeyTransformation
+          prefix: "segments."
+      - type: DpathFlattenFields
+        field_path: ["campaign_budget"]
+        delete_origin_value: True
+        key_transformation:
+          type: KeyTransformation
+          prefix: "campaign_budget."
+      - type: AddFields
+        fields:
+          - path: ["metrics.interaction_event_types"]
+            value: >-
+              {{ record['metrics.interaction_event_types']
+                 | map('replace','UNSPECIFIED','0')
+                 | map('replace','UNKNOWN','1')
+                 | map('replace','CLICK','2')
+                 | map('replace','ENGAGEMENT','3')
+                 | map('replace','VIDEO_VIEW','4')
+                 | map('replace','NONE','5')
+                 | list }}
+    incremental_sync:
+      type: DatetimeBasedCursor
```

Comment:
Was there a reason why we're not specifying a `step` and `cursor_granularity`? Looking at `IncrementalGoogleAdsStream.stream_slices()`, there is the section where for each `customer`, we perform `chunk_data_range()`. And in that block it looks like the `slice_duration` maps to our low-code `step` field and the `slice_step` maps to our low-code `cursor_granularity` field.